source/components/simpleCardList/simpleCard.ts

```diff
@@ -136,6 +136,7 @@ export class SimpleCardController implements ISimpleCardBindings {
 	private updateAlwaysOpen(alwaysOpen: boolean): void {
 		if (alwaysOpen) {
 			this.showContent = true;
+			this.alwaysOpen = true;
```

Comment:
Angular will set this normally, so may want to just update this in the test before calling $onChanges instead.